share/spack/gitlab/cloud_pipelines/configs/multi-src-mirrors.yaml.in

```diff
@@ -0,0 +1,16 @@
+mirrors:
+  buildcache-source:
+    fetch: ${PROTECTED_MIRROR_FETCH_DOMAIN}/${PR_TARGET_REF_NAME}/${SPACK_CI_STACK_NAME}
+    push: s3://spack-binaries/${PR_TARGET_REF_NAME}/${SPACK_CI_STACK_NAME}
```

Comment:
Could this be `${PROTECTED_MIRROR_PUSH_DOMAIN}`? Then we could switch between staging/production from one place.